website/blog/2022-01-20-initial-setup.md

```diff
@@ -20,6 +20,10 @@ To create Eightshift theme, navigate to `wp-content/themes` in your project and
 npx create-wp-project
 ```
 
+Just follow the instructions and wait for the theme to download.
+
+![](/eightshift-docs/img/blog/setup-1.png)
```

Comment:
Add some alt text to this image (the text in the [] should be shown as the alt text I think)